front_end/panels/ai_chat/ui/settings/providers/GenericProviderSettings.ts

```
@@ -0,0 +1,300 @@
+// Copyright 2025 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import { BaseProviderSettings } from './BaseProviderSettings.js';
```

ESLint

[error] There should be at least one empty line between import groups

(import/order)